### PR TITLE
improve/fix manually-moving-data-folders.adoc

### PR DESCRIPTION
**Pull request by voroyam**
still work in progress but a few improvements

### CODE REVIEW

**Comment by mmattel** [comment]
There were some conflicts raised by other PR's from @jnweiger I solved here.
I took the chance to improve some text things additionally.

File: modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc [hunk 1]
```diff
@@ -153,3 +157,28 @@ To fix the config.php settings:
 ----
 {occ-command-example-prefix} config:system:set --value /mnt/owncloud/data datadirectory
 ----
+
+NOTE: You have to repeat this for all apps found defining the data directory as key.
```
Comment (voroyam) [inline modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc:161]: don't we have to remove this note?

Comment (mmattel) [reply modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc:161]: Agreed, this must be removed.

File: modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc [hunk 2]
```diff
@@ -56,16 +53,25 @@ To enable maintenance mode, run the following command.
 {occ-command-example-prefix} maintenance:mode --on
 ----
 
-To disable maintenance mode of your instance run the following command:
+== Sync your Data directory
```
Comment (mmattel) [inline modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc:56]:
```suggestion
== Sync your Data Directory
```

File: modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc (hunk 1 above)
Comment (mmattel) [inline modules/admin_manual/pages/maintenance/manually-moving-data-folders.adoc:161]:
```suggestion
```

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
LGTM, many thanks 👍 
Backport to 10.13 and 10.12

**Pull request merged**